dart/lib/src/transport/noop_transport.dart

```diff
@@ -0,0 +1,17 @@
+import 'dart:async';
+
+import 'package:sentry/sentry.dart';
+
+class NoOpTransport implements Transport {
+  @override
+  Dsn get dsn => null;
+
+  @override
+  String get origin => null;
+
+  @override
+  String get sdkIdentifier => null;
```

Comment:
this should not be part of the public API, they should only care about the `send` method

Reply:
I just let `dsn` for now as it is used in a lot of tests